tests/pipelines/eks/small/upstream-load.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: eks-small-upstream-load-pipeline-cron
+spec:
+  schedule: "0 8 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: curl
+            image: curlimages/curl
+            args: ["curl", "-X", "POST", "--data", "{}", "el-eks-small-upstream-load-cron-listener.svc.cluster.local:8080"]
+          restartPolicy: Never
```

Comment:
let's cut down on spec bloat wherever we can, including here.

Reply:
this is mandatory, explained offline.